packages/web-components/fast-components/src/accordion-item/index.ts

```diff
@@ -13,10 +15,38 @@ import { accordionItemStyles as styles } from "./accordion-item.styles";
  * @remarks
  * HTML Element: \<fast-accordion-item\>
  */
-export const fastAccordionItem = AccordionItem.compose({
+export const fastAccordionItem = AccordionItem.compose<AccordionItemOptions>({
     baseName: "accordion-item",
     template,
     styles,
+    collapsedIcon: html`
```

Comment:
The default option can be a simple string here, which would be more performant in the end. That doesn't prevent the end user from overriding it with a custom template. Since there doesn't appear to be an need for template features, I would just remove the `html` tag from the front of all of these.

Reply:
```suggestion
    collapsedIcon:`
```

Reply:
Thanks @EisenbergEffect - updated to just leverage strings here rather than templates. I think this is a good start for the default scenarios. Over time we can add or update additional components as well.